packages/authorization-process/open-api/authorization-service-spec.yml

```diff
@@ -0,0 +1,1159 @@
+openapi: 3.0.3
+
+info:
+  title: Security Process Micro Service
+  description: This service is the security supplier
+  version: "{{version}}"
+  contact:
+    name: API Support
+    url: http://www.example.com/support
+    email: support@example.com
+  termsOfService: "http://swagger.io/terms/"
+  x-api-id: an x-api-id
+  x-summary: an x-summary
+servers:
+  - url: "/authorization-process/{{version}}"
+    description: This service is the security supplier
+security:
+  - bearerAuth: []
+tags:
+  - name: client
+    description: Get security information
+    externalDocs:
+      description: Find out more
+      url: http://swagger.io
+  - name: user
+    description: Get security information
+    externalDocs:
+      description: Find out more
+      url: http://swagger.io
+  - name: health
+    description: Verify service status
+    externalDocs:
+      description: Find out more
+      url: http://swagger.io
+paths:
+  /clientsConsumer:
+    parameters:
+      - $ref: "#/components/parameters/CorrelationIdHeader"
+    post:
+      tags:
+        - client
+      summary: Create a new consumer client
+      description: Create a new consumer client
+      operationId: createConsumerClient
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/ClientSeed"
+      responses:
+        "200":
+          description: Client created
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Client"
+        "401":
+          description: Unauthorized
+          content:
+            application/problem+json:
```

Comment:
`application/problem+json` is not supported by the `openapi-zod-client` library, this must be converted to `application/json`

Reply:
Updated, thanks